app/views/tasks/show.html.haml

```diff
@@ -48,11 +49,18 @@
         %tbody.govuk-table__body
           %tr.govuk-table__row
             %td.govuk-table__cell
-              Uploaded file
+              = @file.filename || 'Uploaded file'
```

Comment:
I'm struggling to see the circumstances under which `@file` would exist but filename would be blank. Is that even possible?

Reply:
I found some submission files which were missing attachment, but none were the `latest_submission` for a Task, so I'm happy to remove this.